lib/shared/store.js

```diff
@@ -122,25 +124,20 @@ const ACTIONS = _.fromPairs(_.map([
 }))
 
 /**
- * @summary Find a drive from the list of available drives
+ * @summary Get available drives from the state
  * @function
- * @private
+ * @public
  *
- * @param {Object} state - application state
- * @param {String} device - drive device
- * @returns {(Object|Undefined)} drive
+ * @param {Object} state - state object
+ * @returns {Object} new state
  *
  * @example
- * const drive = findDrive(state, '/dev/disk2');
+ * const drives = getAvailableDrives(state)
+ * _.find(drives, { device: '/dev/sda' })
  */
-const findDrive = (state, device) => {
-  /* eslint-disable lodash/prefer-lodash-method */
-
-  return state.get('availableDrives').find((drive) => {
-    return drive.get('device') === device
-  })
-
-  /* eslint-enable lodash/prefer-lodash-method */
+const getAvailableDrives = (state) => {
```

Comment:
I dunno how all the store-related classes are supposed to interact, but is this a "duplicate" of https://github.com/resin-io/etcher/blob/master/lib/shared/models/available-drives.js#L68 ?

Reply:
Yup, but `availableDrives` relies on `store` so we can't expose that, and I don't think we want to expose anything else from the `store` either